src/main/resources/com/nike/cerberus/migration/V1.6.2.0__make_user_groups_case_sensitive.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE USER_GROUP CHANGE NAME NAME VARCHAR(255) BINARY NOT NULL;
```

Comment:
My understanding of this is that changing `NAME` to binary will force the database to use the values of the bytes in the string to compare.

Where I assume, before, all letter casing would be the same regardless of the byte letter values?

Reply:
Yes, this will now allow `foo` and `Foo` and `fOo` to be considered different values and not cause collisions in the index as we did with iam_roles.